charts/semantic-hub/Chart.yaml

```diff
@@ -26,8 +26,8 @@ sources:
   - https://github.com/eclipse-tractusx/sldt-semantic-hub
 
 type: application
-version: 0.1.30
-appVersion: 0.2.12
+version: 0.1.32
```

Comment:
we need to keep the old version of (helm) version i.e  0.1.31, since it is not yet created.

Reply:
Thanks for the review comments @agg3fe . I have incorporated the review comment.